### DISCUSSION TITLE
Bad dumps (probably soldering error?)

### DISCUSSION THREAD

**Discussion by kamalawala** [sanni/cartreader#553] [Need help]
I've finally gotten a good deal of soldering done to where I can test with GBX games, however it seems I'm getting bad dumps and I feel that the issue could be 1 or 2 bad solder joints. I don't have a multimeter to check for continuity but I can provide some bad dumps, if that narrows it down at all.
Attached is Namco Museum (USA) (GBA) and Street Fighter 2 (Europe) (GB)

**Comment by sanni** [comment]
Check if there is a solder bridge connecting D1 and D2 on the GB slot or the pin header below.

![nfxbnxbxbbxbxbxbx](https://user-images.githubusercontent.com/19169494/194779936-104ee7d1-41a7-47bf-90ce-a5dfe34e9d78.jpg)

**Reply by kamalawala** [reply]
Thank you!! I think D1 and D2 on the Game Boy port itself were the problem. Getting perfect dumps now. Love the project! Don't have any non-GBX games to try yet, so I'm content.
